examples/Swift/Sample/TailLoadingCellNode.swift

```diff
@@ -38,6 +39,9 @@ final class TailLoadingCellNode: ASCellNode {
   }
 
   override func layoutSpecThatFits(constrainedSize: ASSizeRange) -> ASLayoutSpec {
+    
+    spinner.style.flexBasis = ASDimensionMakeWithPoints(0.0)
```

Comment:
As the node is wrapping a UIView component it cannot provide it's own intrinsic content size, so for flexing we need to give a minimum size to it.